README.md

```diff
@@ -194,107 +206,107 @@ You can now pass JWTs from Apple to the `verify` method. The key identifier (`ki
 
 HMAC is the simplest JWT signing algorithm. It uses a single key that can both sign and verify tokens. The key can be any length.
 
-- `hs256`: HMAC with SHA-256
-- `hs384`: HMAC with SHA-384
-- `hs512`: HMAC with SHA-512
+To add an HMAC key to the key collection, use the `addHS256`, `addHS384`, or `addHS512` methods:
 
 ```swift
 // Add HMAC with SHA-256 signer.
-signers.use(.hs256(key: "secret"))
+await keys.addHS256(key: "secret")
```

Comment:
I'm interested to see what people think of this API - it's the only thing that seems a bit less Swifty